fusequery/query/src/configs/config.rs

```diff
@@ -31,87 +31,76 @@ lazy_static! {
     pub log_level: String,
 
-    #[structopt(long, env = "FUSE_QUERY_LOG_DIR", default_value = "./_logs")]
+    #[structopt(long, default_value = "./_logs")]
     pub log_dir: String,
 
-    #[structopt(long, env = "FUSE_QUERY_NUM_CPUS", default_value = "0")]
+    #[structopt(long, default_value = "0")]
     pub num_cpus: u64,
 
-    #[structopt(
-        long,
-        env = "FUSE_QUERY_MYSQL_HANDLER_HOST",
-        default_value = "127.0.0.1"
-    )]
+    #[structopt(long, default_value = "127.0.0.1")]
     pub mysql_handler_host: String,
 
-    #[structopt(long, env = "FUSE_QUERY_MYSQL_HANDLER_PORT", default_value = "3307")]
+    #[structopt(long, default_value = "3307")]
     pub mysql_handler_port: u16,
 
-    #[structopt(
-        long,
-        env = "FUSE_QUERY_MYSQL_HANDLER_THREAD_NUM",
-        default_value = "256"
-    )]
+    #[structopt(long, default_value = "256")]
     pub mysql_handler_thread_num: u64,
 
-    #[structopt(
-        long,
-        env = "FUSE_QUERY_CLICKHOUSE_HANDLER_HOST",
-        default_value = "127.0.0.1"
-    )]
+    #[structopt(long, default_value = "127.0.0.1")]
     pub clickhouse_handler_host: String,
 
-    #[structopt(
-        long,
-        env = "FUSE_QUERY_CLICKHOUSE_HANDLER_PORT",
-        default_value = "9000"
-    )]
+    #[structopt(long, default_value = "9000")]
     pub clickhouse_handler_port: u64,
 
-    #[structopt(
-        long,
-        env = "FUSE_QUERY_CLICKHOUSE_HANDLER_THREAD_NUM",
-        default_value = "256"
-    )]
+    #[structopt(long, default_value = "256")]
     pub clickhouse_handler_thread_num: u64,
 
-    #[structopt(
-        long,
-        env = "FUSE_QUERY_FLIGHT_API_ADDRESS",
-        default_value = "127.0.0.1:9090"
-    )]
+    #[structopt(long, default_value = "127.0.0.1:9090")]
     pub flight_api_address: String,
 
-    #[structopt(
-        long,
-        env = "FUSE_QUERY_HTTP_API_ADDRESS",
-        default_value = "127.0.0.1:8080"
-    )]
+    #[structopt(long, default_value = "127.0.0.1:8080")]
     pub http_api_address: String,
 
-    #[structopt(
-        long,
-        env = "FUSE_QUERY_METRIC_API_ADDRESS",
-        default_value = "127.0.0.1:7070"
-    )]
+    #[structopt(long, default_value = "127.0.0.1:7070")]
     pub metric_api_address: String,
 
-    #[structopt(long, env = "STORE_API_ADDRESS", default_value = "127.0.0.1:9191")]
+    #[structopt(long, default_value = "127.0.0.1:9191")]
     pub store_api_address: String,
 
-    #[structopt(long, env = "STORE_API_USERNAME", default_value = "root")]
+    #[structopt(long, default_value = "root")]
     pub store_api_username: String,
 
-    #[structopt(long, env = "STORE_API_PASSWORD", default_value = "root")]
+    #[structopt(long, default_value = "root")]
     pub store_api_password: String,
 
-    #[structopt(long, short = "c", env = "CONFIG_FILE", default_value = "")]
+    #[structopt(long, short = "c", default_value = "")]
     pub config_file: String,
 }
 
+// adaptor for environment variable
```

Comment:
Hello,
Why not based on the `Config`?
In this case, we need to maintain two configurations